src/features/import/components/ImportDialog/Configure/Configuration/EnumConfigRow.tsx

```diff
@@ -0,0 +1,98 @@
+import { ArrowForward, Delete } from '@mui/icons-material';
+import {
+  Box,
+  FormControl,
+  IconButton,
+  InputLabel,
+  MenuItem,
+  Select,
+  Typography,
+} from '@mui/material';
+import { FC } from 'react';
+
+import messageIds from 'features/import/l10n/messageIds';
+import { EnumChoice } from 'utils/types/zetkin';
+import { Msg, useMessages } from 'core/i18n';
+
+interface EnumConfigRowProps {
+  italic?: boolean;
+  numRows: number;
+  onSelectOption: (key: string) => void;
+  onDeselectOption: () => void;
+  options: EnumChoice[];
+  selectedOption: string | null;
+  title: string;
+}
+
+const EnumConfigRow: FC<EnumConfigRowProps> = ({
```

Comment:
I see that this is a copy of the `OrgConfigRow`, but without the `mapping` state. I can see why that state might seem redundant, especially when you're testing with just a few choices in an enum.

But the reason we introduced it in the org column was that there can sometimes be hundreds of organizations and when React tried to create hundreds of rows, with hundreds of `Select` options in each, the Browser would stall for a few seconds.

So we added the state so that the rows are all created with no `Select`, and the `Select` is only rendered once the user has chosen to actively map a certain value.

I think the same thing is relevant here, especially since enum fields will often be used for organizations as well. So I think it might make sense to use the `mapping` state (and it's logic) here as well.

What do you think?

Reply:
Ah I understand, but it does seem like a redundant step when not needed. Maybe some limit, say 10 options?

Reply:
It's not a bad idea, but lets not make it more complicated right now and instead just use the same solution as for organizations. We can come back (to both) and make improvements in the future.

Reply:
Ok, fixed